docs/user-guide/title-section.md

```diff
@@ -83,6 +106,4 @@ The background toolbar, in this case, changes a little bit by including an addit
 
 <img src="../img/title-section/map-toolbar.jpg" class="ms-docimage" style="width:300px"/>
 
-* the **Edit map configuration** through which is possible to [Configure the Map](configure-map.md)
-
-
+* The **Edit map configuration** through which is possible to [Configure the Map](configure-map.md)
```

Comment:
* The **Edit map configuration** through which it is possible to [Configure the Map](configure-map.md)